commercetools-sdk-base/src/main/java/io/sphere/sdk/client/UnauthorizedException.java

```diff
@@ -1,5 +1,10 @@
 package io.sphere.sdk.client;
 
+/**
+ * Unauthorized access to the commercetools platform wither through invalid client credentials or tokens.
```

Comment:
Ah nice. Although it's hard to read if you don't know it. Maybe between quotes? "wither"